openpower/package/hostboot-binaries/hostboot_binaries.mk

```diff
@@ -5,8 +5,8 @@
 ################################################################################
 
 
-HOSTBOOT_BINARIES_VERSION ?= f36e9a3ea2a5e8d733fd801518e62de076261f07
-HOSTBOOT_BINARIES_SITE ?= $(call github,open-power,hostboot-binaries,$(HOSTBOOT_BINARIES_VERSION))
+HOSTBOOT_BINARIES_VERSION ?= eeb88fc5f80d2e49af525cad02a28f2b06f3769c
+HOSTBOOT_BINARIES_SITE ?= $(call github,jahunsbe,hostboot-binaries,$(HOSTBOOT_BINARIES_VERSION))
```

Comment:
Will need to get this merged in hostboot-binaries and put this back to open-power